cmd/kubeless/function/call.go

```diff
@@ -19,18 +19,28 @@ package function
 import (
 	"bytes"
 	"fmt"
+	"math/rand"
```

Comment:
`UnixNano()` only gives us a few bytes of entropy between competing controllers that start at similar times, which will lead to more frequent event id collisions than we otherwise expect.  I suggest using `crypto/rand` instead, to draw on an a better entropy source.